.github/workflows/build_release.yml

```
@@ -0,0 +1,173 @@
+
+      - name: Generate checksum and define release date
+        run: |
+          CHECKSUM=$(sha256 -q "artifacts/${{ env.PKG_DOT_NAME }}.pkg")
+          echo "${CHECKSUM} ${{ env.PKG_DOT_NAME }}.pkg" > "artifacts/${{ env.PKG_DOT_NAME }}.pkg.sha256"
+          echo "CHECKSUM=${CHECKSUM}" >> "$GITHUB_ENV"
+          RELEASE_DATE="$(stat -f %Sm -t "%Y-%m-%d %H:%M:%S" "artifacts/${{ env.PKG_DOT_NAME }}.pkg")"
+          echo "RELEASE_DATE=${RELEASE_DATE}" >> "$GITHUB_ENV"
+
+      - name: Get latest release
+        id: latest
+        uses: pozetroninc/github-action-get-latest-release@v0.8.0 # v0.8.0
```

yamllint

[warning] too few spaces before comment

(comments)